dispatch/modules/content/embeds.py

```diff
@@ -78,7 +78,7 @@ def render(self, data):
         try:
             return tag(tags[data['mode']], data['content'])
         except KeyError:
-            return data['content']
+            return '<div class="o-article-embed o-article-embed--video"> <div class="o-article-embed--video__wrapper">' + data['content'] + '</div></div>'
```

Comment:
Instead of reusing the video embed wrapper, can you create a new class specifically for code embeds?

```html
<div class="o-article-embed o-article-embed--code">
```

You should then be able to get away with only using one `div` instead of two